content/swiftui/concepts/viewmodifier/terms/strikethrough/strikethrough.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.strikethrough()'
+Description: 'Applies a horizontal line through the middle of the text in a string.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.strikethrough`** view modifier method in SwiftUI can be used to strike through text within a view.
+> **Note**: This method is specifically for use with text, so it can be applied as a view modifier to any view that displays text, such as a Text or Label View. However, it cannot be applied to views that do not display text, such as an Image View.
```

Comment:
```suggestion

> **Note**: This method is specifically for use with text, so it can be applied as a view modifier to any view that displays text, such as a Text or Label View. However, it cannot be applied to views that do not display text, such as an Image View.
```